appsmithorg/appsmith
# fix: changed the editing to false on onFocus in PrimaryColumnsControl

## Description

Column name in the property of the table widget duplicates when dragging the columns from top to bottom or from bottom to top.

By setting the state `setEditing` to false on `onFocus` solved the issue, since it will stop the editing of the input while dragging.

Fixes #9188 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

I have tested this manually. 

Instructions to test:

1. Click one of the rows of the column property in the property pane of the table widget
2. With mouse still being pressed down, drag the row down.
3. Once the row is dragged to the desired position release the mouse button.
4. Observe now that the column sequence is changed correctly both in the table and the column property in the property pane.

Please find below the video for the reference:
[![LOOM DEMO](http://cdn.loom.com/sessions/thumbnails/766f8a2d7e2246eab6558d27ee1c754f-00001.gif)](https://www.loom.com/share/766f8a2d7e2246eab6558d27ee1c754f)

## Checklist:







## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>


    // Code coverage diff between base branch:release and head branch: fix/fix-column_name_order_change_table 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | app/client/src/selectors/commentsSelectors.ts | 85.25 **(1.64)** | 64.71 **(2.95)** | 73.33 **(0)** | 90.59 **(2.35)**
 :red_circle: | app/client/src/utils/WorkerUtil.ts | 88.98 **(-0.78)** | 70.59 **(-1.96)** | 100 **(0)** | 92.38 **(-0.95)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.71 **(-0.23)** | 40.83 **(-0.84)** | 36.21 **(0)** | 56.74 **(-0.25)**</details>
